pkg/querier/blocks_store_queryable.go

```diff
@@ -562,7 +562,6 @@ func (q *blocksStoreQuerier) fetchSeriesFromStores(
 	minT int64,
 	maxT int64,
 	matchers []*labels.Matcher,
-	convertedMatchers []storepb.LabelMatcher,
```

Comment:
Why do we change this?

Reply:
Cause i wanna convert after removing the shard label.

Reply:
I see. I am not sure about the benefits but it is okay to me.

Reply:
Discussed on slack. We need to remove it before sending to store-gateways.